features/in_foreground.feature

```diff
@@ -1,7 +1,7 @@
 Feature: In foreground field populates correctly
 
 Scenario: Test handled exception after delay
-    When I run "InForegroundScenario"
+    When I run "HandledExceptionScenario"
```

Comment:
For this scenario, we just need to verify that when a regular exception is thrown in the foreground - this is achievable by using `HandledExceptionScenario` since we altered `InForegroundScenario` elsewhere.